api/api.go

```diff
@@ -53,7 +55,7 @@ func New(addressBook *peers.AddressBook, dht *dht.DHT, exchange nnet.Exchange, b
 	streamsEnd := router.Group("/api/v1/streams")
 	streamsEnd.GET("/:ns/*pattern", api.HandleGetStreams)
 
-	router.LoadHTMLFiles("./api/index.html")
+	router.Use(ServeFs("/", Assets))
```

Comment:
Gin has the `router. StaticFS` have you tried that?

Reply:
I have use https://github.com/jessevdk/go-assets-builder and it plays well with the `router.StaticFS` and its kind of what they suggest in gin.

Reply:
Any reason you chose the other one?

Reply:
Other one?